upgrade/helpers/tasks.py

```diff
@@ -2167,6 +2167,11 @@ def pulp_migration_monitoring_after_timeout():
     return False
 
 
+def pulp2_removal():
+    output = run('satellite-maintain content remove-pulp2 -y --plaintext')
```

Comment:
Doc string is missing

Reply:
Oops. I've hurried towards working upgrade pipelines.
Going to add it. Thanks

Reply:
Docstring added.